src/components/Charts/zoneUtils.js

```diff
@@ -0,0 +1,156 @@
+import { COLOR_MAP } from '../../enums/interventions';
```

Comment:
I think this files is A+ should we move these to be a common enum and share also with the status indciator component?

Reply:
yes - I moved the definitions to enums, following your example from https://github.com/covid-projections/covid-projections/pull/599